book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
```

Comment:
- [ ] It would be better if you have a method for adding rentals, Kindly check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). 

```ruby
def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```